examples/graph_data_demo.ipynb

```diff
@@ -139,17 +174,168 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "source": [
+    "## Save and Load\n",
```

Comment:
let's follow the pattern of other notebooks and make these more like phrases:
`Saving and Loading a Profile`